modules/apps/27-interchain-accounts/types/codec_test.go

```diff
@@ -156,3 +307,172 @@ func (suite *TypesTestSuite) TestDeserializeAndSerializeCosmosTxWithAmino() {
 	suite.Require().Error(err)
 	suite.Require().Empty(bz)
 }
+
+func (suite *TypesTestSuite) TestJSONDeserializeCosmosTx() {
+	var cwBytes []byte
+	var protoMessages []proto.Message
+	testCases := []struct {
+		name     string
+		malleate func()
```

Comment:
I think in this table test, since `malleate` is only assinging `cwBytes` and `protoMessages`, it'd be nicer if the fields were directly in the test case, but just a suggestion

Reply:
done